[Workers] add wrangler update instructions (#5005)
cloudflare/cloudflare-docs: content/workers/wrangler/get-started.md

weight: 1
---

> ## Get started with Wrangler

Reviewer: ```suggestion
# Get started with Wrangler
```

> Wrangler is a command line tool (CL) for building [Cloudflare Workers](https://workers.cloudflare.com/).

Reviewer: ```suggestion
Wrangler is a command-line tool (CL) for building [Cloudflare Workers](https://workers.cloudflare.com/).
```
Should this be `command-line interface (CLI)` instead? Not sure if we use `CL` elsewhere.

{{<Aside type="note">}}

Wrangler has launched a new version. If you are looking for the Wrangler 1 documentation, you can find it [here](/workers/wrangler/cli-wrangler/install-update/).

If you previously had Wrangler 1 installed or were working on a Wrangler 1 project, refer to the [Migration guide](/workers/wrangler/migration/migrating-from-wrangler-1/).

{{</Aside>}}

## Install

> Installing `wrangler`, the Workers CLI, allows you to [`init`](/workers/wrangler/commands/#init), [`dev`](/workers/wrangler/commands/#dev), and [`publish`](/workers/wrangler/commands/#publish) your Workers projects.

Reviewer: ```suggestion
Installing Wrangler, the Workers CLI, allows you to [`init`](/workers/wrangler/commands/#init), [`dev`](/workers/wrangler/commands/#dev), and [`publish`](/workers/wrangler/commands/#publish) your Workers projects.
```
Not sure about this one... feel free to ignore.

To install [`wrangler`](https://github.com/cloudflare/wrangler2), ensure you have [`npm` installed](https://www.npmjs.com/get-npm), preferably using a Node version manager like [Volta](https://volta.sh/) or [nvm](https://github.com/nvm-sh/nvm). Using a version manager helps avoid permission issues and allows you to easily change Node.js versions. Then run:

```sh
$ npm install -g wrangler
```

or install with `yarn`:

```sh
$ yarn global add wrangler
```

To check your Wrangler version, run:

```sh
$ wrangler --version
```

{{<Aside type="note">}}

If you do not want to install Wrangler globally, you can use the [npx](https://docs.npmjs.com/cli/v8/commands/npx) command. For example: `npx wrangler init`.

{{</Aside>}}

## Authenticate

> With installation complete,Wranglerwill need access to a Cloudflare OAuth token to manage Workers resources on your behalf.

Author: ```suggestion
With installation complete, Wrangler will need access to a Cloudflare OAuth token to manage Workers resources on your behalf.
```

Open the browser, log in to your account, and select **Allow**. This will send an OAuth Token to Wrangler so it can deploy your scripts to Cloudflare.

> ## Update

Author: @rozenmd

Reviewer 2: Yep that looks correct 👍

Reviewer 2: Actually @deadlypants1973, @petebacondarwin suggests `npm -g install wrangler@latest` instead of `npm update -g wrangler`

To update Wrangler, run:

```sh

> $ npm update -g wrangler

Author: ```suggestion
$ npm -g install wrangler@latest
```
